rs/bitcoin/kyt/src/main.rs

```diff
@@ -127,8 +146,103 @@ fn post_upgrade(arg: KytArg) {
 #[ic_cdk::query(hidden = true)]
 fn http_request(req: http::HttpRequest) -> http::HttpResponse {
     if req.path() == "/metrics" {
```

Comment:
should we maybe add a metric related to response sizes so that we can get some distribution, something similar to what was done [here](https://github.com/dfinity/ic/blob/80d6ecdc421047ef36acbbca5ea52cef0188acb9/rs/ethereum/cketh/minter/src/eth_rpc.rs#L860)?